### PR TITLE
Test media tree using v4l2 with the vivid driver

### PR DESCRIPTION
**Pull request by ana**
Add a virtualvideo config fragment, this will enable the Virtual Video  Test Driver (vivid) and add a template variant to update the v4l2 test plan to run it on QEMU.

The last commit enables sending the media tree emails reports to linux-media@, ezequiel and guillaume.

### CODE REVIEW

File: test-configs.yaml [hunk 1]
```diff
@@ -120,6 +120,12 @@ test_plans:
   v4l2:
```
Comment (gctucker) [inline test-configs.yaml]: The commit subject "test-configs: run v4l2 with the stretch-v4l2 rootfs" should be changed to show it's for QEMU, as for other devices the v4l2 test plan is already run on stretch-v4l2.

File: jenkins/kernel-arch-complete.sh [hunk 2]
```diff
@@ -209,3 +209,23 @@ if [[ BUILDS_FINISHED -eq 4 ]]; then
         fi
     fi
 fi
+
+# We're only building the media tree in arm, arm64 and x86
```
Comment (gctucker) [inline jenkins/kernel-arch-complete.sh]: While this was true on the media-pilot branch, it's not the case any more in production.  So this "media" tree case should be moved to the list above like all the other trees.  Also I think we can send both plain text and html version of the reports.

File: jenkins/kernel-arch-complete.sh [hunk 3]
```diff
@@ -209,3 +209,23 @@ if [[ BUILDS_FINISHED -eq 4 ]]; then
...
+    if [ -f ${BASEDIR}/arm64.done ] && [ -f ${BASEDIR}/arm.done ] && [ -f ${BASEDIR}/x86.done ]; then
+
+        echo "Build has now finished, reporting result to dashboard."
+        curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'"}' ${API}/job
+        if [ "$EMAIL" != "true" ]; then
+            echo "Not sending emails because EMAIL was false"
+            exit 0
+        fi
+        if [ "$BRANCH" == "master" ]; then
+            curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'", "build_report": 1, "send_to": ["linux-media@vger.kernel.org", "ezequiel@collabora.com", "guillaume.tucker@collabora.com"], "format": ["txt"], "delay": 10}' ${API}/send
```
Comment (gctucker) [inline jenkins/kernel-arch-complete.sh]: I don't really need to have my email address here, I'll get the reports from the mailing list.  This was mostly useful for testing things on the media-pilot branch.

**Comment by ana** [comment]
Thanks @gctucker. I have updated the PR with these changes.

**Comment by gctucker** [comment]
Tested OK on staging.

**Pull request merged**